### DISCUSSION TITLE
[Suggestion] (For Planet Surfaces) Gas Giants

### DISCUSSION THREAD

**Discussion by Jorpis-Jorah** [tsunamayo/Starship-EVO#4092] [Suggestions]
NOTE: This suggestion was originally made on steam by WILDCATreactor. They asked, and I quote:

"so, instead of a killzone, will we eventually get a damage field down past the "normal pressure" layer of the atmosphere (probably about the same time collision damage gets implemented, haha) I'd love to see people make cloud cities and explore the upper sections of super storms"

They suggested that in the future when planets are added, we could "Land" on gas giants and I thought this was a good idea.
Star citizen recently released its own gas giant in 3.14, but im very certain that Unity wouldn't be able to render clouds at stellar distances, so here are some rendering techniques that could be used:

2d to 3d Clouds, As you fly closer to the gas giant, clouds could go from 2d clouds to 3d counterparts, subtle.

Lod radius, similar to Asteroids, closer you get, the more the lod is increased.

i'm not sure if a gas giant is possible in Unity but it's worth a shot.
Cheers!

**Comment by Jorpis-Jorah** [comment]


**Reply by TIKIRobo** [reply]
The issues section is ment for bugs while suggestions go here

**Comment by Jorpis-Jorah** [comment]
So...could this be done?

**Comment by Rusty-uy** [comment]
i would love to see that

**Comment by TE60** [comment]
gas giants could have floating landmasses, or maybe some sort of floating moss that you can land on
